src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss

```diff
@@ -1,6 +1,20 @@
 @import "./../../../../assets/stylesheets/custom";
 @import "./../../../../assets/stylesheets/mixins";
 @import "./../../../../assets/stylesheets/color-variables";
+.separator{
```

Comment:
All this css classes needs to be in .detail-container as per their hierarchy level